static/src/stylesheets/module/atoms/_snippets.scss

```diff
@@ -0,0 +1,66 @@
+.explainer-snippet__image {
+    float: left;
+    margin-right: $gs-gutter / 2;
+
+    display: inline-block;
+    position: relative;
+    width: 33%;
+    height: 0;
+    overflow: hidden;
+    border-radius: 50%;
+    padding-top: 33%;
```

Comment:
Could you make this use `$gs-baseline`? In case there is a reason for the percentages, just ignore the comment :)